mingw-w64-mcfgthread/PKGBUILD

```
@@ -26,8 +26,8 @@ build() {
   [[ -d "${srcdir}/build-${MSYSTEM}" ]] && rm -rf "${srcdir}/build-${MSYSTEM}"
   mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
 
-  export CFLAGS+=' -Os -g'
   MSYS2_ARG_CONV_EXCL="--prefix=" meson setup -Dbuildtype=plain  \
+    -Dc_flags='-Os -g' -Dc_link_flags='-Os'  \
```

```suggestion
    -Dc_args='-Os -g' -Dc_link_args='-Os'  \
```
but I don't know whether it rewrites CFLAGS environment variable either appends to it